src/libuv/Native.Settings

```diff
@@ -37,11 +35,13 @@
     <ClCompile>
       <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
       <Optimization>Disabled</Optimization>
+      <GenerateDebugInformation>true</GenerateDebugInformation>
```

Comment:
Symbols are still generated, they probably aren't as specific. This isn't compatible with the control flow guard